### ISSUE TITLE
Deactivating Decent Land

### ISSUE THREAD

**Issue by allquantor** [labscommunity/arweavekit#92]
ans-stats.decent.land/profile continuously checks for the ans domain once you connect your wallet.

Could we have a config flag to deactivate it? It does polute the developer console and is an unnecessary request in some cases.

**Comment by 7i7o** [comment]
Thanks for the feedback @allquantor
I assume you meant to add this to the [`arweave-wallet-kit`](https://github.com/labscommunity/arweave-wallet-kit) repo instead of this one, but we'll add it to the roadmap there anyway. Thanks!

**Comment by allquantor** [comment]
Yes. Thanks for the correction @7i7o. Do you want me to close and repost over there?
